packages/application-shell/src/components/user-settings-menu/user-settings-menu.tsx

```diff
@@ -70,17 +69,18 @@ const UserAvatar = (
   const handleMouseOut = useCallback(() => {
     setIsMouseOver(false);
   }, []);
-  const { theme } = useTheme();
+  const { isNewTheme, themedValue } = useTheme();
   return (
     <div onMouseOver={handleMouseOver} onMouseOut={handleMouseOut}>
       <Spacings.Inline alignItems="center">
         <Avatar
+          size={themedValue('s', 'l')}
```

Comment:
The correct size

Reply:
cc @FilPob

Reply:
sorry, I dont know what and where is being changed here

Reply:
Avatar size

Reply:
Size `l` is too big, I changed it to `m`.

Size `s` (current)

<img width="498" alt="image" src="https://user-images.githubusercontent.com/1110551/224312287-f8c7a92c-1ee7-4c9f-b646-f23f40591114.png">


Size `m`

<img width="487" alt="image" src="https://user-images.githubusercontent.com/1110551/224312196-e6496be5-9869-4041-aa82-3dea7ed49743.png">